chia/full_node/coin_store.py

```diff
@@ -435,12 +436,13 @@ async def rollback_to_block(self, block_index: int) -> List[CoinRecord]:
                     record = CoinRecord(coin, uint32(0), row[1], row[2], uint64(0))
                     coin_changes[record.name] = record
 
-            # Delete from storage
+            # Delete reverted blocks from storage
             await conn.execute("DELETE FROM coin_record WHERE confirmed_index>?", (block_index,))
```

Comment:
shouldn't coins that disappear in the reorg also be reported as "changed" to the wallet?

Reply:
Yeah those are collected starting in line 429.